schema-engine/connectors/sql-schema-connector/src/warnings/generators.rs

```diff
@@ -275,6 +279,15 @@ pub(crate) struct ModelAndIndex {
     pub(crate) index_db_name: String,
 }
 
+/// An constraint in a model that triggered a warning.
+#[derive(Serialize, Debug, Clone)]
+pub(crate) struct ModelAndConstraint {
+    /// The name of the model
+    pub(crate) model: String,
+    /// The name of the index
+    pub(crate) constraint: String,
+}
```

Comment:
I'm a little confused here as the struct is `ModelAndConstraint` and the field is `constraint` but the doc is saying name of the _index_. These are not necessarily the same thing right

Reply:
Copy&pasta. It should say constraint.